web_hosting_manager/app/utils/app_utils.js

```diff
@@ -4,6 +4,29 @@ export const trimErrorMsg = (msg) => {
   return msg.slice(index).trim()
 };
 
+export const parseErrorMsg = (err) => {
+  switch(err.code) {
+    case -102: {
+      return 'Account already exist';
+    }
+    case -107: {
+      return 'Entry already exist';
+    }
+    case -108: {
+      return 'Mutable data maximum size reached';
```

Comment:
I think we should be showing an interpretation of the error to the user, this message is what internally happened but to the user of this application that doesn't mean anything, I believe he intention was to translate this errors to something meaningful to the user, e.g. in this case we could say 'No more services allowed for current public ID' or 'No more files can be added to current service', etc. So there is a need of a context for some of these errors as in this case since it depends on which mutable data we were operating with.